[improvement]complement the SHOW PROCESSLIST information

dh-cloud opened the pull request:
The SHOW PROCESSLIST Statement displays information about the processes that are currently running in the server.
but the information is incomplete, so we add some information: query_id, sql detail and query start time.

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (No Need)
3. Has document been added or modified: (No Need)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

jackwener commented on fe/fe-core/src/main/java/org/apache/doris/analysis/ShowProcesslistStmt.java:
```diff
@@ -32,10 +32,12 @@
                     .addColumn(new Column("Time", ScalarType.createType(PrimitiveType.INT)))
                     .addColumn(new Column("State", ScalarType.createVarchar(64)))
-                    .addColumn(new Column("Info", ScalarType.createVarchar(16)))
+                    .addColumn(new Column("Info", ScalarType.createVarchar(32 * 1024)))
```
This change of size need to be discussed. @EmmyMiao87

jackwener commented on fe/fe-core/src/main/java/org/apache/doris/qe/ConnectContext.java:
```diff
@@ -549,12 +551,24 @@ public String getRemoteIp() {
             row.add("" + (nowMs - startTime) / 1000);
             row.add("");
-            row.add("");
+            String stmt = "";
+            if (executor != null){
+                stmt = executor.getOriginStmtInString();
+            }
```
```suggestion
            String stmt = executor == null ? "" : executor.getOriginStmtInString();
```

dh-cloud replied:
changed

jackwener reviewed (commented):
Thanks for contribution ❤.

jackwener commented:
BTW, you can change the title following the [guidance](https://doris.apache.org/zh-CN/community/how-to-contribute/commit-format-specification.html#commit-%E6%A0%BC%E5%BC%8F%E8%A7%84%E8%8C%83)

dh-cloud committed:
complement the SHOW PROCESSLIST information

The SHOW PROCESSLIST Statement displays information about the processes that are currently running in the server.
but the information is incomplete, so we add some information: query_id, sql detail and query start time.

EmmyMiao87 commented on fe/fe-core/src/main/java/org/apache/doris/qe/ConnectContext.java:
```diff
@@ -549,12 +551,21 @@ public String getRemoteIp() {
             row.add("" + (nowMs - startTime) / 1000);
             row.add("");
-            row.add("");
+            String stmt = executor == null ? "" : executor.getOriginStmtInString();
```
We can already view the currently running query through the function of ```SHOW PROC "/current_query_stmts"```. 
What is the main purpose of adding here?

dh-cloud replied:
we can view current sql already running time,  can use the query_id to find log , for locate the perform problem, like postgresql: select * from pg_stat_activity
